code/world.dm

```diff
@@ -304,3 +302,12 @@
 		s += ": [jointext(features, ", ")]"
 
 	status = s
+
+/world/proc/update_hub_visibility(new_visibility)
+	if(new_visibility == GLOB.hub_visibility)
+		return
+	GLOB.hub_visibility = new_visibility
+	if(GLOB.hub_visibility)
+		hub_password = "kMZy3U5jJHSiBQjr"
+	else
+		hub_password = "SORRYNOPASSWORD"
```

Comment:
Also, it seems that tgstation has a "defines syndrome" where they try to use defines for everything

The AIR1 and AIR2 and AIRI defines are absolutely useless and made by someone licking chalkboard erasers.